src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs

```diff
@@ -158,6 +164,60 @@ static void ConfigureFilters(ILoggerFactory factory)
 #else
 				.AddConsole(LogLevel.Information);
 #endif
+		}
+
+#if __ANDROID__
+
+        Android.Content.BroadcastReceiver androidBroadcastReceiver = new AndroidBroadcastReceiverForUWPTriggers();
+
+
+	#region "handling TimeTrigger"
+
+    // for API 21 (), since 2014
+    [Android.App.Service(Permission = "android.permission.BIND_JOB_SERVICE")]
```

Comment:
This cannot be added by default in the Uno templates. Can this be implemented in Uno as UWP background service normal declaration ?

Reply:
But what if two apps has same service?
Is Android.App.Service full name (namespace) be inside app namespace, or in Uno namespace?

Reply:
I think - not.

Reply:
This won't be added to the templates by default. We'll need to find another way to support this.

Reply:
Why we can't make new templates, and throw exception from code if template is not "current version"?

I don't really know internal Uno structure, so I cannot think up any solution (other than implemented in this PR).

Reply:
Can I mark this conversation as resolved?

Reply:
Can I mark this conversation as resolved?

Reply:
Can I mark this conversation as resolved?

Reply:
No, the original comment has not been resolved. If this portion of the code is not meant to be modified by the app developer, it should be inside of Uno.